repo2docker/buildpacks/julia.py

```diff
@@ -22,9 +31,30 @@ def get_env(self):
         ]
 
     def get_path(self):
+        """Get path for Julia executables
+
+         Returns:
```

Comment:
Adds path to Julia binaries to user's PATH.

Also we can simplify this code to just be:

```
return super().get_path() + ['${JULIA_HOME}']
```

but also ok to not do that!